### ISSUE TITLE
[ItemRegistrationIssue]Error: java.lang.IllegalArgumentException: Failed to create model for minecraft:pig

### ISSUE THREAD

**Issue by xFirefalconx** [konwboj/Tumbleweed#33]
Hi, first of all, I have no clue why Tumbleweed got told to cause this, but I thought I send a report anyways. 

[crash-2023-02-01_14.42.57-client.txt](https://github.com/konwboj/Tumbleweed/files/10558390/crash-2023-02-01_14.42.57-client.txt)

For some reason this first showed up with a mod called "CherryBlossomGrotto" and I deactivated it, started, played the game. But after I added a mod called "Haunted Harvest" and tried to start, it appeared again, even after deactivating said first mod. Now it says Tumbleweed is the culprit? 

I also sent a report to pehkui because it also showed up in the crash report relatively early. 
If you got any idea who else might be causing this crash, I will also send them a bug report if I can. I just want to play... sigh. 

Forge version: 40.2.0 (necessary for some other mods) (I have about 189 mods in our modpack as of now).

Thanks for your help in advance.

**Comment by xFirefalconx** [comment]
Edit: When I deactivated Tumbleweed, it started again. It might have to do with the other mods I'm using because I kept having Tumbleweed in my mods for a long while now and this is the first crash.

I don't know if it's a pehkui incompatibility bug or it got to do with Haunted Harvest?

**Comment by xFirefalconx** [comment]
**Versions**
Minecraft Version: 1.18.2
Minecraft Forge Version: 40.2.0
Mod Version:  0.5.4

**Log files**
Please append the log, debug-log and if available the crash-log files.

[latest.log](https://github.com/GeheimagentNr1/ManyIdeasDoors/files/10663977/latest.log)
[2023-02-06-7.log.gz](https://github.com/GeheimagentNr1/ManyIdeasDoors/files/10663978/2023-02-06-7.log.gz)
[crash-2023-02-06_14.07.02-client.txt](https://github.com/GeheimagentNr1/ManyIdeasDoors/files/10663981/crash-2023-02-06_14.07.02-client.txt)
[crash-2023-02-06_14.11.17-client.txt](https://github.com/GeheimagentNr1/ManyIdeasDoors/files/10663982/crash-2023-02-06_14.11.17-client.txt)
[crash-2023-02-06_13.32.04-client.txt](https://github.com/GeheimagentNr1/ManyIdeasDoors/files/10663983/crash-2023-02-06_13.32.04-client.txt)
[crash-2023-02-06_14.05.21-fml.txt](https://github.com/GeheimagentNr1/ManyIdeasDoors/files/10663985/crash-2023-02-06_14.05.21-fml.txt)
Not all may be relevant but I included them to see the full magnitude. 
Modpack of 190 mods. There MIGHT be EVEN MORE culprits among these as I am still to test 57 mods that potentially can be innocent, though. Please do get in touch with the other authors linked below if you are not a cause but a bystander. 

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behaviour:
1. Add following mods: SakuraBlossomGrotto (+ Terra Blender), this also happens with Tumbleweed, add Astikor's Carts. The game will crash when trying to create the pig model due to a NullPointerReferenceExeption. 
Disable either Astikor's Carts or both SakuraBlossomGrotto and Tumbleweed. If either AC, MID + SBG or T are together, the game crashes.

**Expected behaviour**
No crash, lol.

**Screenshots**
If applicable, add screenshots to help explain your problem.
![grafik](https://user-images.githubusercontent.com/106652912/216996205-a1be2451-a2dd-40dd-bb90-68744ba4a4d2.png)


**Additional context**

Other crash reports on the other mods
https://github.com/Dekunutter/CherryBlossomGrotto/issues/36 
https://github.com/konwboj/Tumbleweed/issues/33 (no response yet)
https://github.com/GeheimagentNr1/ManyIdeasDoors/issues/4
https://github.com/issork/astikor-carts/issues/88

**Comment by xFirefalconx** [comment]
Dekunutter found a reson for the crash. It has to do with the way items are registered. Please read the thread and see if you can also implement a fix on your side. https://github.com/Dekunutter/CherryBlossomGrotto/issues/36 In hope you find a way, there is an explanation on what can be done, I am just a noob you might understand it better. 

Tumbleweeds is currently incompatible to ManyIdeasDoors + AstikorCarts combined. Sakurablossomgrotto was incompatible but is now fixed. I hope you can be so kind to apply this to your mod as well so I can activate all mods. 🙏

**Comment by konwboj** [comment]
It's an AstikorCarts issue. Posted a comment there.

**Comment by whatisagoodusername5256** [comment]
Didn't want to make a new thread when one already existed, but the mod Rubidium gives the same thing related to "Failed to create model for minecraft:pig" when using this mod with it. Not sure if I should make a new thread for this instead with more details or comment on the Github page for Rubidium instead.

**Comment by konwboj** [comment]
I think Rubidium has a greater chance of being the culprit

**Comment by xFirefalconx** [comment]
Thank you for your aid in finding the AC crash issue, konboj. 
Verph could fix it. I already commented in the AC github but I wanted to let you know that this was also thanks to your investigation, I contacted them after collecting all the info and that lead to enough clues that it could be nailed down onto the function that was carrying the exact line you mentioned in their issue [#88](https://github.com/issork/astikor-carts/issues/88). 
I made a fork of Verph's changes and published a band-aid version. Both mods get along with each other now :) This would perhaps not been possible without everyone's help.
